pkg/activitypub/service/inbox/inbox_test.go

```diff
@@ -300,15 +299,13 @@ func TestInbox_Error(t *testing.T) {
 		undeliverableChan, err := pubSub.Subscribe(context.Background(), spi.UndeliverableTopic)
 		require.NoError(t, err)
 
-		var mutex sync.Mutex
-
 		var undeliverableMessages []*message.Message
 
+		done := make(chan struct{})
```

Comment:
I believe that I fixed the race condition in MockPubSub that was causing this issue (in yesterday's PR).

Reply:
yes